File: InventoryDemo.DTOs/Requests/PersonnelPropertyAddRequest.cs
﻿using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace InventoryDemo.DTOs.Requests
{
    public class PersonnelPropertyAddRequest
    {
        public int PropertyID { get; set; }
        public int UserID { get; set; }
        public DateTime DueOn { get; set; }
        public bool IsWaiting { get; set; }
    }
}

File: frontend/lib/Models/PersonnelProperty/PersonnelPropertyAddRequest.dart
class PersonnelPropertyAddRequest {
  final int personnelID;
  final int propertyID;
  final DateTime dueOn;
  final bool isWaiting;

  const PersonnelPropertyAddRequest(
      {required this.personnelID,
      required this.propertyID,
      required this.dueOn,
      required this.isWaiting});
}
